falcon/request.py

```diff
@@ -524,6 +534,12 @@ def cookies(self):
         return self._cookies.copy()
 
     @property
+    def json(self):
```

Comment:
Also when converting to method as @philiptzou suggested then there is no need for storing value in additional attribute for later reference (responsibility of a caller) and thus additional slot.